Website/src/rendering/src/components/SpeakersGrid.tsx

```diff
@@ -1,74 +1,44 @@
+  };
+};
 
-const SpeakersGrid = (): JSX.Element => (
-  <div className="section__speakers__grid">
-    <Link href="/speakers/mary-asada">
-      <a className="section__speakers__grid__speaker">
-        <Image src={speaker1} alt="Speaker" width={260} height={260} />
-        <p className="speaker__name">Mary Asada</p>
-        <p className="speaker__role">Athlete</p>
-      </a>
-    </Link>
-    <Link href="/speakers/martinmoore">
-      <a className="section__speakers__grid__speaker">
-        <Image src={speaker2} alt="Speaker" width={260} height={260} />
-        <p className="speaker__name">Martin Moore</p>
-        <p className="speaker__role">Speaker</p>
-      </a>
-    </Link>
-    <Link href="/speakers/jacobgonzalez">
-      <a className="section__speakers__grid__speaker">
-        <Image src={speaker3} alt="Speaker" width={260} height={260} />
-        <p className="speaker__name">Jacob Gonzalez</p>
-        <p className="speaker__role">Speaker</p>
-      </a>
-    </Link>
-    <Link href="/speakers/edjones">
-      <a className="section__speakers__grid__speaker">
-        <Image src={speaker4} alt="Speaker" width={260} height={260} />
-        <p className="speaker__name">Ed Jones</p>
-        <p className="speaker__role">Speaker</p>
-      </a>
-    </Link>
-    <Link href="/speakers/sophiataylor">
-      <a className="section__speakers__grid__speaker">
-        <Image src={speaker5} alt="Speaker" width={260} height={260} />
-        <p className="speaker__name">Sophia Taylor</p>
-        <p className="speaker__role">Speaker</p>
-      </a>
-    </Link>
-    <Link href="/speakers/lixiuying">
-      <a className="section__speakers__grid__speaker">
-        <Image src={speaker6} alt="Speaker" width={260} height={260} />
-        <p className="speaker__name">Li Xiu Ying</p>
-        <p className="speaker__role">Speaker</p>
-      </a>
-    </Link>
-    <Link href="/speakers/kategreen">
-      <a className="section__speakers__grid__speaker">
-        <Image src={speaker7} alt="Speaker" width={260} height={260} />
-        <p className="speaker__name">Kate Green</p>
-        <p className="speaker__role">Speaker</p>
-      </a>
-    </Link>
-    <Link href="/speakers/jalenwilliams">
-      <a className="section__speakers__grid__speaker">
-        <Image src={speaker8} alt="Speaker" width={260} height={260} />
-        <p className="speaker__name">Jalen Williams</p>
-        <p className="speaker__role">Speaker</p>
-      </a>
-    </Link>
-  </div>
-);
+type SpeakersGridProps = ComponentProps & {
+  fields: {
+    items: Speaker[];
+  };
+};
+
+const SpeakersGrid = (props: SpeakersGridProps): JSX.Element => {
+  console.log(props.fields);
+
+  return (
+    <div className="section__speakers__grid">
+      {props.fields.items &&
+        props.fields.items.map((speaker, index) => {
+          return (
+            <Link
+              key={index}
+              href={'/speakers/' + speaker.fields.Name.value.replace(/ /g, '')}
```

Comment:
I am unsure but maybe LayoutService is including the URL of the sub items that have a layout. It could be used directly instead of crafting the URL yourself.

Reply:
not sure how to do this. putting it on the back burner